src/redux/services/compare/transferResource.ts

```diff
@@ -100,12 +101,12 @@ async function deployResourceToCluster(
   }
 
   const id = target?.id ?? uuid();
-  const resource = createResource(updatedContent, {
+  const resource = createResource(updatedContent, 'cluster', {
     id,
     origin: {
       storage: 'cluster',
       context: currentContext,
-    },
+    } as ClusterOrigin,
```

Comment:
I only added the `as ClusterOrigin` cast to the override - since this method both creates and deploys a resource it overrides the transient storage set by `createResource`